aiaccel/hpo/apps/optimize.py

```diff
@@ -112,26 +114,25 @@ def main() -> None:
 
     parser = argparse.ArgumentParser()
     parser.add_argument("job_filename", type=Path, help="The shell script to execute.")
-    parser.add_argument("--config", nargs="?", default=None)
+    parser.add_argument("--config", help="Configuration file path")
     parser.add_argument("--executor", nargs="?", default="local")
     parser.add_argument("--resume", action="store_true", default=False)
+    parser.add_argument("--resumable", action="store_true", default=False)
 
     args, unk_args = parser.parse_known_args()
-    config = oc.merge(oc.load(args.config), oc.from_cli(unk_args))
 
-    if "storage" not in config.study:
-        config.study.storage = {
-            "_target_": "optuna.storages.RDBStorage",
-            "url": "sqlite:///optuna.db",
-            "engine_kwargs": {"connect_args": {"timeout": 30}},
-        }
+    default_config = oc.load(importlib.resources.open_text("aiaccel.hpo.apps.config", "default.yaml"))
+    config = default_config if args.config is None else oc.merge(default_config, oc.load(args.config))
```

Comment:
細かいですが，この方がロジック見えやすいかなと思いました．
```suggestion
    config = oc.merge(default_config, oc.load(args.config) if args.config is not None else {})
```